modules/virt-configuring-node-exporter-service.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-configuring-node-exporter-service_{context}"]
+= Configuring the node exporter service
+
+The `node-exporter` agent is deployed on every node and collects metrics about every node in a cluster. Configure the `node-exporter` agent as a service to expose metrics from virtual machines.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in to the cluster as a user with `cluster-admin` privileges.
+* Create the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+* Configure the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project by adding `enableUserWorkload` and setting its value to `true`.
+
+.Procedure
+
+. Create the `node-exporter` service yaml file.
++
+[source,yaml]
+----
+kind: Service
+apiVersion: v1
+metadata:
+  name: node-exporter-service <1>
+  namespace: dynamation <2>
+  labels:
+    servicetype: metrics <3>
+spec:
+  ports:
+    - name: exmet
+      protocol: TCP
+      port: 9100 <4>
+      targetPort: 9100 <5>
+  type: ClusterIP
+  selector:
+    monitor: metrics <4>
+----
+<1> The `node-exporter` service that exposes the metrics from the virtual machines.
+<2> Namespace where the service is created.
+<3> Label for the selector that applies to the `node-exporter` service. This label is used by the `ServiceMonitor` to match the service.
+<4> TCP port number of the virtual machine that is configured with the `servicetype` label.
+<5> Target port of the `ClusterIP` that exposes the metrics.
+
```

Comment:
add a number 6 here, see previous comment

Reply:
Updated the content.